src/main/resources/locales/cy/translation/single-or-multiple-claim.json

```diff
@@ -1,13 +1,14 @@
 {
```

Comment:
I have put these as blank so that if there isn't a welth translation it will display the english as a fallback, if I put "welsh translation required" in values then we will display that in the front end which I want to avoid.

Further enchancements we can make to these tests is to add tests that fail if there is missing translations